content/blog/k8s-122-challenge/index.md

```diff
@@ -0,0 +1,51 @@
+---
+path: '/k8s-122-challenge'
+title: 'Unexpected Challenges Supporting Kubernetes 1.22 in Pixie'
+date: 2021-10-05T06:00:00.000+00:00
+featured_image: k8s122challenge.png
```

Comment:
Can you rename this file to `hero-image.png` to be consistent with the other posts? And compress it using tinypng.com or some other tool?